packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -191,6 +214,69 @@ export function buildRootObject(powers, _params, testJigSetter = undefined) {
       return { zcfSeat, userSeat: userSeatPromiseKit.promise };
     };
 
+    /** @type {MakeInvitation} */
+    const makeInvitation = (
+      offerHandler = () => {},
+      description,
+      customProperties = {},
+    ) => {
+      assert.typeof(
+        description,
+        'string',
+        details`invitations must have a description string: ${description}`,
+      );
+
+      const invitationHandle = makeHandle('InvitationHandle');
+      invitationHandleToHandler.init(invitationHandle, offerHandler);
+      /** @type {Promise<Payment>} */
+      const invitation = E(zoeInstanceAdmin).makeInvitation(
+        invitationHandle,
+        description,
+        customProperties,
+      );
+      return invitation;
+    };
```

Comment:
`makeInvitation` should call `makeInvitationWithSeat` and just drop the seat. We shouldn't have two pathways